src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +17,30 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> arr = new ArrayList<>();
+
+        File file = new File(fileName);
+        if (file.length() == 0) {
+            return new String[]{};
+        }
+        Path path = Paths.get(fileName);
+        try {
+            List<String> array = Files.readAllLines(path);
+            for (String value : array) {
+                String[] tmp = value.toLowerCase().split(" ");
+                for (String s : tmp) {
+                    if (s.startsWith("w")) {
```

Comment:
make "w" constant of a class

Reply:
> 
> 
> make "w" constant of a class

I will also make regex constant, is it fine?

Reply:
as you wish :) it's optional